staging/src/k8s.io/apiserver/pkg/util/flowcontrol/request/config.go

```diff
@@ -23,8 +23,8 @@ import (
 )
 
 const (
-	minimumSeats                = 1
-	maximumSeats                = 10
+	minimumSeatsLimit           = 1
```

Comment:
I do not see why "Limit" is being added to the name of "minimumSeats".  I understand the change from "maximumSeats" to "maximumSeatsLimit" --- this is a change from a fixed maximum to a bound on the maximum.  But the minimum is staying static, there is no need to bound its variation.

Reply:
hmm -- this is just to say consistent with `maximumSeatsLimit`. In theory it is still a limit, but one we don't change. If we choose to change this later or expose it as an external config it would make more sense as `minimumSeatsLimit`.

Reply:
I don't feel strongly about the naming of this to be honest, can change it back to just `minimumSeats`

Reply:
Worth noting that I started with removing this field entirely and just using `1` everywhere since we don't change minimumSeats (and I don't forsee us trying to dynamically change min  seats in the future).  https://github.com/kubernetes/kubernetes/pull/118601#discussion_r1262049705 

(I agree with adding the  constant back though, so we don't forget places to update minSeats if it does change for any reaosn)

Reply:
Yes, I saw the history here.
I do not think it is a big deal here, not a blocker, but in my opinion it is unnecessary complexity to talk about a limit on a static value.

Reply:
Naming these two things according to the same pattern is positively confusing.  One is a static bound.  The other is a bound on a variable bound.  One is a first-order bound, the other is a second-order bound.  Naming them both like a second-order bound makes the reader think they are both second-order bounds.